src/util/ChartUtils.ts

```diff
@@ -1272,23 +1280,23 @@ export const getStackedDataOfItem = <StackedData>(
   return null;
 };
 
-const getDomainOfSingle = (data: Array<Array<any>>): number[] =>
-  data.reduce(
-    (result: number[], entry: Array<any>): number[] => [
-      min(entry.concat([result[0]]).filter(isNumber)),
-      max(entry.concat([result[1]]).filter(isNumber)),
-    ],
-    [Infinity, -Infinity],
-  );
+const getDomainOfSingle = (data: Array<Array<any>>): number[] => {
```

Comment:
`getDomainOfSingle` what? Ik this is preexisting but maybe we can add a comment here

Reply:
Of single stack. I will delete this soon.

Reply:
Cool cool